app/labor/cache_buster.rb

```diff
@@ -19,17 +22,55 @@ def self.bust(path)
     #
     # https://github.com/fastly/fastly-ruby#efficient-purging
     return unless Rails.env.production?
-    return if ApplicationConfig["FASTLY_API_KEY"].blank?
 
-    HTTParty.post("https://api.fastly.com/purge/https://#{ApplicationConfig['APP_DOMAIN']}#{path}",
-                  headers: { "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"] })
-    HTTParty.post("https://api.fastly.com/purge/https://#{ApplicationConfig['APP_DOMAIN']}#{path}?i=i",
-                  headers: { "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"] })
+    if fastly_enabled?
+      bust_fastly_cache(path)
+    elsif nginx_enabled?
+      bust_nginx_cache(path)
+    else
+      raise ConfigurationError, "You cannot bust a cache without a caching service! Please enable a caching service."
```

Comment:
Let me know if this feels too aggressive; my thought here was that we don't want to try to _ever_ call `bust` on a cache if we don't have _some_ kind of caching service configured, but maybe I'm wrong about that. 🤷